content/docs/dvclive/api-reference/index.md

```diff
@@ -25,6 +25,12 @@ from dvclive import Live
 live = Live()
 ```
 
+Or use as a context manager:
+
+```python
+with Live() as live:
+```
+
```

Comment:
Not blocking (but probably would block if I wasn't trying to get this out today): can we remove the suggested usage without the context manager? It could result in unexpected behavior.

IMO there are 2 categories of people who will use it without the context manager:
1. People who can figure out themselves that they can omit it and understand the consequences of doing so (they don't need this documented).
2. People who won't be able to figure out this pattern for themselves but may also be surprised at the consequences (they probably should use the context manager).

Reply:
Also, can we avoid mentioning `context manager`? I think it's too technical.

Reply:
Makes sense.